### DISCUSSION TITLE
more css options as in vite

### DISCUSSION THREAD

**Discussion by igortas** [farm-fe/farm#994] [Ideas]
Hi,

I like to see ```css``` option where I can configure various options related to ```css```, like minify and transformation.

```vite``` have that option, and lets u use another tool than the default one...

Here is example how I'm using it in ```vite```, lightning css:

```
  build: {
    outDir: 'build',
    cssMinify: 'lightningcss'
  },
  css: {
    transformer: 'lightningcss',
    lightningcss: {
      targets: browserslistToTargets([
        'last 1 chrome version',
        'last 1 firefox version',
        'last 1 safari version'
      ])
    }
  }
```

Farm already have the ```browserlist option``` so it's good if we can select other options like ```lightning css``` as opt-in solution

BR,

**Comment by wre232114** [comment]
Advanced css tools like `lightningcss` should be supported by a plugin in Farm. Farm use `swc_css` for css `parsing`, `transforming` in pure rust for best performance, checkout css config [css.prefixer.targets](https://farm-fe.github.io/docs/config/compilation-options#cssprefixertargets) for internally supported css features.

I think we need to implement a `farm-plugin-lightningcss` to satisfy your needs, we'll try to implement it later, or PR welcome.
